util/file.go

```diff
@@ -218,6 +217,17 @@ func GetPathRelativeTo(path string, basePath string) (string, error) {
 	return filepath.ToSlash(relPath), nil
 }
 
+func GetPathRelativeToWithSeparator(path string, basePath string) (string, error) {
+	relPath, err := GetPathRelativeTo(path, basePath)
+	if err == nil {
+		if relPath == "." {
+			relPath += string(filepath.Separator)
```

Comment:
Why do we want the trailing slash? I'm not saying it's wrong, just that `.` is as useful to me as `./`.